out/ackermann_desc.json

```diff
@@ -31,7 +31,7 @@
     "sources": [
-        "D:\\code\\scryptlib\\test\\fixture\\ackermann.scrypt"
+        "C:\\Users\\myland\\code\\scryptlib\\test\\fixture\\ackermann.scrypt"
```

Comment:
Why did this change? Let's try minimizing unnecessary changes from now on, if not this time.

Reply:
previeus commit is by my book, now is by my pc. so path change.

Reply:
no change